docs/design.ltx

```diff
@@ -0,0 +1,357 @@
+
+After the training stage, AVIRIS images with unknown land surfaces will be processed with the trained algorithm. The algorithm will classify each pixel into a class representing a land surface and that data will be used for further analysis (see design items \ref{subsec:identifymining}, \ref{subsec:preprocessing}, and \ref{subsec:correlate}) and possibly be output into an image to be used on the homepage. 
+
+Something to consider when training and using this algorithm for analysis is spectral separability. Some types of minerals and other land surfaces have very similar relationships between reflectance and wavelength and can be cause the algorithm’s accuracy to suffer if they are separated into different classes \cite{separability}. It is up to the developers to determine if the particular minerals or other land surfaces that are not very separable should be put into the same class or kept separate depending on if the minerals or other land surfaces in question are important enough by themselves in some way to warrant a slightly lower accuracy.
+
+
+% xiaomei: requirement 3.4.2: identify mining
+\subsection{Identify Mining}
+\label{subsec:identifymining}
+
+After we get the mineral identification and classification data, we need to identify the regions of mining activity. The data is further processed and possibly combined with geographic information about geology and mines. The goal of identifying mining is to accurately locate regions in which mining is taking place in a form that is suitable for both human inspection and further processing. This step creates input data for further processing of the project and the data evaluation.
+
+To start our process of identifying mining, we need the output from the mineral identification step. The previous step has already located the regions that are containing mineral associated with mining. To distinguish mines from preexisting geology is a challenge; only rely on the existing geography data is not enough for the identification.
+
+There are three methods mentioned after several group discussion sessions for identifying mining: mineral classification, using patterns of mineral deposits; GIS comparison, using geographic information such as mining permit boundaries or geologic maps; and a combination of mineral classification and GIS comparison, using both sources of data. To locate regions with mineral deposits that indicate mining, we have decided that we are going to use the mineral classification methods. Then, we would move to a combination of mineral classification and GIS comparison methods once we are able to complete the first step without much trouble.  
```

Comment:
I believe the three methods @xiaomei7 mentioned are referred to in the corresponding section of the technology review that I wrote. I was using "mineral classification" in a broad sense to refer to the kind of classification done by the spectral library. It would make sense to cite it explicitly.

If I recall, my comments about geological maps were proposed as a way of reducing false positives from classification alone: For example, if we identify a large region of gravel but know that it is a natural riverbed, then we could use the GIS data to exclude that match. Probably overkill for our purposes, but worth considering if we want to process that kind of data.